dash-pipeline/bmv2/dash_acl.p4

```diff
@@ -35,7 +36,8 @@ match_kind {
     @name(str(table_name##:dash_acl_rule|dash_acl)) \
     table table_name { \
         key = { \
-            meta. ## table_name ##_dash_acl_group_id : exact @name("meta.dash_acl_group_id:dash_acl_group_id"); \
+            meta. ## table_name ##_dash_acl_group_id : exact @name("meta.dash_acl_group_id:dash_acl_group_id") \
+            @sai[type="sai_object_id_t", isresourcetype="true", objects="SAI_OBJECT_TYPE_DASH_ACL_GROUP"]; \
```

Comment:
Please use @Sai. Per P4-16 spec, custom annos should start with upper case to distinguish from built-ins.

Reply:
Done